x/auction/keeper/abci_test.go

```diff
@@ -11,16 +12,16 @@ func (suite *KeeperTestSuite) TestAbci() {
 	ctx, auctionKeeper := suite.ctx, suite.auctionKeeper
 	require := suite.Require()
 
-	params := auctionTypes.DefaultParams()
-	auctionKeeper.setParams(ctx, params)
+	auctionParams := auctionTypes.DefaultParams()
+	auctionKeeper.setParams(ctx, auctionParams)
 
 	// Test base case of no auctions
 	// Note BeginBlocker is only run once for completeness, since it has no code in it
 	require.NotPanics(func() { auctionKeeper.BeginBlocker(ctx) })
 	require.NotPanics(func() { auctionKeeper.EndBlocker(ctx) })
 
 	// Create an auction
-	sommPrice := auctionTypes.TokenPrice{Denom: auctionTypes.UsommDenom, UsdPrice: sdk.MustNewDecFromStr("0.01"), LastUpdatedBlock: 5}
+	sommPrice := auctionTypes.TokenPrice{Denom: params.BaseCoinUnit, UsdPrice: sdk.MustNewDecFromStr("0.01"), LastUpdatedBlock: 5}
```

Comment:
Nit: in this refactor, we're repeating the thing of importing as `params` vs. `appParams` to avoid potentially overloading a value. I see here that you accounted for this by renaming this particular params retrieval `auctionParams`, but just a note as a general concept that this introduces a potential headache.